app/metrics.yaml

```diff
@@ -62,6 +62,31 @@ events:
     notification_emails:
     - telemetry-client-dev@mozilla.com
     expires: "2020-03-01"
+  crash_reporter_open:
+    type: event
+    description: >
+      The crash reporter was displayed
+    bugs:
+    - 1040
+    data_reviews:
+    - TBD
+    notification_emails:
+    - telemetry-client-dev@mozilla.com
```

Comment:
Still need to update this email, mentioning @vesta0. Or is there a bug where this is handled so I can just reference instead of pinging all the time? 😅

Reply:
#1157 :)